src/build.rs

```diff
@@ -271,6 +264,50 @@ impl FieldsBuilder<UnnamedFields> {
 	}
 }
 
+/// Build a field.
+pub struct FieldBuilder<T> {
+	name: Option<&'static str>,
+	ty: MetaType,
+	ty_display_name: Option<Path>,
+	marker: PhantomData<fn() -> T>,
+}
+
+impl<T> FieldBuilder<T> {
+	/// Create a new field builder for a field with the given [`MetaType`].
+	pub fn new(ty: MetaType) -> Self {
+		Self {
+			name: None,
+			ty,
+			ty_display_name: None,
+			marker: PhantomData,
+		}
+	}
+
+	/// Specify the display name of the type of the field. This can either be the type name itself
```

Comment:
Why not name it "type alias"? "Display" seems like something used in a UI, but I don't think that's the only use case here. Can these be anything but type aliases?

Reply:
Because it applies to any field type name, not only an alias.